### DISCUSSION TITLE
Application of extreme dependency scores

### DISCUSSION THREAD

**Discussion by jwraby** [dtcenter/METplus#1920] [Statistical Computation] [answered]
Referring to the following article: 
"Stephenson, D.B., B. Casati, C.A.T. Ferro, and C.A. Wilson, 2008: The extreme
dependency score: A non-vanishing measure for forecasts of rare events.
Meteorological Applications 15, 41-50.

Can this method be applied to forecasts of other variables besides precipitation? I'm thinking of continuous variables such as wind speed or temperature.

Thanks.

**Comment by jprestop** [comment, marked as answer]
Hi @jwraby.  I've reached out to @ericgilleland and @bgbrowntollerud for assistance with this answer.

**Reply by ericgilleland** [reply]
Hi @jwraby Yes, the score can be applied to continuous variables.  It is categorical, so you will need to assign appropriate thresholds.

**Reply by jwraby** [reply]
Thanks Julie and Eric!
